src/sparseml/pytorch/utils/quantization/helpers.py

```diff
@@ -98,16 +97,24 @@ class QATWrapper(Module):
     :param reduce_range: if True, the quantization range will be reduced by one bit.
         This may prevent overflow issues with model execution on certain hardware
         Default is False
+    :param activation_kwargs: Additional kwargs for quantization of activations.
+        Default is {}
     """
 
     @staticmethod
-    def from_module(module: Module, reduce_range: bool = None) -> "QATWrapper":
+    def from_module(
+        module: Module,
+        reduce_range: bool = None,
+        activation_kwargs: Dict[str, Any] = {},
```

Comment:
nit: rename to `activation_qconfig_kwargs` since it's just for the qconfigs

Reply:
also need to add a param for the weights

Reply:
Added!